_posts/2021-06-04-flavourzyme-a-purified-enzyme-mixture-from-a-oryzae.md

```diff
@@ -0,0 +1,103 @@
+
+#### 2. Filamentous fungi make everything viscous 
+
+Filamentous fungi don't normally grow in a submerged environment. The mycelia will aggregate to form pellets or clumps - which reduces substrate uptake and enzyme secretion. To complicate things, a bioreactor full of filamentous mycelia will make the mixture more and more viscous. This makes stirring harder, and also reduces the ability of air bubbles to mix and for oxygen to dissolve. By necessity, oxygen transfer drops as the mycelia grows. 
+
+#### 3. Use a pulsed-paused feeding mode 
+
+In addition to using a fed-batch setup, researchers have also found that pulsed-paused feeding, in which feeding of the substrate is turned on/off intermittently in 300s cycles, improves the fungi's morphology. The rationale is that under starvation considerations, fungal mycelium are known to autolyse (self-digest) older parts of the mycelium network to re-allocate resources to the growing tip. While this mechanism is yet to be studied in detail, it's been shown that you get better morphology and lower viscosity with pulsed-paused feeding, without sacrificing enzyme productivity [4]. 
+
+Once the upstream process is finished, you end up with a crude liquid containing fungi, enzymes, spent substrate, and all sorts of metabolites.
+
+### Refining the enzymes - downstream processing
+
+In *downstream processing*, we want to refine out the enzymes only. While I couldn't find any literature specifically on the downstream processing of flavourzyme, reading textbook on enzymes was enough to cover the basics [5]. Then, a producer of a Flavourzyme-like product on Alibaba just gave me their HACCP flow diagram, so that made things easy. Here's one example of how it can be done:
+
+#### 1. Filter Press 
+
+The first step is to filter out the mycelia, cell debris, any any other large solids. A *filter press* is used here with perlite, followed by diatomaceous earth, as a filtering aid. The solid matter is captured in the filter aid and removed as filter cake - leaving behind an enzyme mixture. 
+
+![Removing large solids from the raw liquid using perlite and diatomaceous earth ](/assets/images/flavourzyme/Deadendfiltration.png)
+*Schematic of dead-end filtration with a filter aid - elements not to scale*
+
+Note that filtration is the first step because we are dealing with **extracellular** enzymes. If the product was an **intracellular** enzyme, this step would be preceded by a 'disruption' step to break apart cells. 
+
+#### 2. Ultrafiltration Concentration 
+
+At this point, our enzyme mixture is still at a fairly low concentration. Another type of filtration called *ultrafiltration* is used to remove water. By passing the liquid across a filter of the appropriate size (in this case, a filter rated around 10kDa), water will permeate through, leaving behind a more concentrated enzyme solution. This can be repeated until the desired concentration is achieved.
```

Comment:
There's no concern for other exo-enzymes in the mixture? ex: some amylase is still present?